backend/chromeApiServer/src/main/java/com/canvas/controllers/chromeApiServer/ChromeApiController.java

```diff
@@ -127,6 +133,51 @@ public ResponseEntity<CommandOutput> initiateStudentCodeEvaluation(
         return evaluation.compileStudentCodeFile(user, files);
     }
 
+    /**
+     * Get request to fetch access token from canvas.
+     *
+     * @param code code returned by canvas after login
+     * @return Redirects to loginSuccess/loginFailure route based on access token response from canvas
+     * @throws IncorrectRequestParamsException
+     */
+    @GetMapping(
+            value = "/oauth2Response"
+    )
+    public ResponseEntity<String> oAuth2Response (
+            @RequestParam(value = "code", required = true) String code) {
+        try {
+            String accessToken = this.oauthService.fetchAccessTokenResponse(code,"http://localhost:8080/oauth2Response");
```

Comment:
The URL needs to be changed to our backend server. 
@fkogge / @gaaliciA1990  could you point me to the right URL?

Reply:
This is updated.